v2/pkg/buildassets/buildassets.go

```diff
@@ -83,6 +80,7 @@ func ReadFileWithProjectData(projectData *project.Project, file string) ([]byte,
 // ProjectInfo if necessary.
 // It will also write the resolved final file back to the project build folder.
 func ReadOriginalFileWithProjectDataAndSave(projectData *project.Project, file string) ([]byte, error) {
+	rootFolder := filepath.Base(projectData.GetBuildDir())
 	file = path.Join(rootFolder, file)
 	content, err := iofs.ReadFile(assets, file)
```

Comment:
Same comment as above, I would say this should be: `content, err := iofs.ReadFile(assets, path.Join("build", file))` where file is the unchanged file argument.